docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+
+services:
+  redis:
+    image: redis
+    container_name: cache
+    expose:
+      - 6379
```

Comment:
Does this expose it to the world, or just other services? I'm not a docker expert, but just want to make sure there are no security holes.

Reply:
I believe that depends on how the users docker network layer is configured. Assuming the default bridge driver, then this will only expose that port to other containers in the same docker network.

Here is a good read on `expose` vs `ports`: https://stackoverflow.com/questions/40801772/what-is-the-difference-between-docker-compose-ports-vs-expose